src/ClientData/FastRenderingUtilsTest.cpp

```diff
@@ -12,6 +12,18 @@ constexpr uint64_t kStartNs = 100;
 constexpr uint64_t kEndNs = 200;
 static std::vector<uint32_t> kPixelResolutionsInTest = {1, 20, 30, 50, 100};
 
+TEST(FastRenderingUtils, GetPixelNumber) {
+  for (uint32_t resolution : kPixelResolutionsInTest) {
```

Comment:
Consider parametrizing the test (https://go/parameterized-testing)

Reply:
Done. Also done for the other method GetNextPixelBoundaryTimeNs.